package.json

```diff
@@ -41,19 +45,20 @@
     "karma-jasmine": "^1.1.0",
     "karma-mocha-reporter": "^2.2.5",
     "karma-source-map-support": "^1.2.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
     "rollupify": "^0.4.0",
     "tslint": "^5.8.0",
     "typescript": "^2.5.3",
     "watchify": "^3.9.0"
   },
   "dependencies": {
-    "@types/angular": ">=1.5.0",
-    "@types/react": "^16.0.18",
-    "@types/react-dom": "^16.0.2",
-    "angular": ">=1.5.0",
     "lodash.kebabcase": "^4.1.1",
-    "ngimport": "^0.7.2",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "ngimport": "^0.7.2"
+  },
+  "peerDependencies": {
+    "angular": ">=1.5.0",
+    "react": "^15.4.2 || ^16.0.0",
```

Comment:
`>= 15.0.0`?

Reply:
yea it could be. it was 15.4.2 before so i just added that in as another option to make the smallest change possible. But if 15.0 is safe then that's fine too.